src/containers/Book.js

```diff
@@ -0,0 +1,23 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+// eslint-disable-next-line arrow-body-style
+const Book = ({ book }) => {
+  const {
+    author, id, title, category,
+  } = book;
+  return (
```

Comment:
- [ ] Since we are using Redux in this project we need to follow the Redux pattern to divide components into Presentational components (not aware of Redux) and Container Components( aware of Redux). You can check [this link](https://redux.js.org/basics/usage-with-react#presentational-and-container-components) to read more about it. The Book component does connect with redux, therefore, should be placed at `src/components`